test/Microsoft.AspNet.Mvc.FunctionalTests/WebApiCompatShimBasicTest.cs

```diff
@@ -275,32 +275,6 @@ public class WebApiCompatShimBasicTest
             Assert.Equal(38, response.Content.Headers.ContentLength);
         }
 
-        [Fact]
-        public async Task ApiController_ResponseReturned_Chunked()
```

Comment:
As discussed, please assume the behavior of IIS/In-memory server is correct and leave this test in place.

We should see:
- A single non-chunked response
- With a set content length
- No `Transfer-Encoding: chunked`